src/Microsoft.ML.Transforms/Text/WordTokenizing.cs

```diff
@@ -397,9 +396,28 @@ private JToken SaveAsPfaCore(BoundPfaContext ctx, int iinfo, JToken srcToken)
     }
 
     /// <summary>
-    /// Word tokenizer splits text into tokens using the delimiter.
-    /// For each text input, the output column is a variable vector of text.
+    /// Tokenizes input text using specified delimiters.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Scalar of [Text](xref:Microsoft.ML.Data.TextDataViewType)  |
```

Comment:
>Scalar [](start = 35, length = 6)

Check IsColumnTypeValid() we don't care about is it scalar or vector as long as underlying type is Text